pkg/apis/apps/v1beta1/register.go

```diff
@@ -57,6 +57,8 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 		&Scale{},
 		&StatefulSet{},
 		&StatefulSetList{},
+		&ControllerRevision{},
```

Comment:
As per decision on call, these are fine to leave in v1beta1, but we'll want to ensure the documentation reflects that we reserve the right to change the beta api.